app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name}"
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
```

Comment:
Using f-strings to construct SQL queries can lead to SQL injection vulnerabilities. Since there are no parameters here, it's safer to directly use the table name without f-string formatting.